src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -103,7 +103,8 @@ def stats(items):
             _items = list(items)
             return {
                 'total': len(_items),
-                'publishable': sum([i.is_publishable() for i in _items])
+                'publishable': sum([i.is_publishable() for i in _items]),
+                'publishable_paths': list(dict.fromkeys([i.path for i in _items if i.is_publishable()]))
```

Comment:
You don't need the `list(dict.fromkeys())` here as you are already building a list with the list comprehension.

Reply:
I'm trying to avoid the duplicates, maybe doing:

```
list(set(mylist))
'publishable_paths': list(set([i.path for i in _items if i.is_publishable()]))
```

Reply:
All right, you can then do : ```list({i.path for i in _items if i.is_publishable()})```